crates/driver/example.toml

```diff
@@ -3,13 +3,19 @@ name = "mysolver" # Arbitrary name given to this solver, must be unique
 endpoint = "http://0.0.0.0:33219/solve"
 absolute-slippage = "12" # Denominated in wei, optional
 relative-slippage = "0.1" # Percentage in the [0, 1] range
-address = "0xE33bff104C4E1140A3f86f855AD5bDAb8Dc6CA8C" # The ETH address of this solver
+address = "0x2cf0a2Fef084b6DB3fb3161c8f2b6B61bcE2F73c" # The ETH address of this solver
+private-key = "0xaa1de59084f3b7e501f2d48cafb2ee04cd06a79ea126fd27ddf3d1b8903bb85a" # The private key of the solver
```

Comment:
From a configuration standpoint, is there a reason we need to specify both address and private key? I would suggest:
- Only specifying the `address` for use with a node that manages your PK (for example, what we would use in CI) or for "dry-run" mode
- Only specify a `private-key` if you want to actually sign transactions.

Reply:
This is correct.